ydb/library/yql/sql/v1/sql_query.cpp

```diff
@@ -1239,8 +1239,12 @@ bool TSqlQuery::Statement(TVector<TNodePtr>& blocks, const TRule_sql_stmt_core&
             }
 
             std::map<TString, TDeferredAtom> features;
-            ParseViewOptions(features, node.GetRule_with_table_settings4());
-            ParseViewQuery(features, node.GetRule_select_stmt6());
+            if (!ParseViewOptions(features, node.GetRule_with_table_settings4())) {
```

Comment:
Here I fix a pretty impactful bug: the results of the view query verification by [building](https://github.com/ydb-platform/ydb/blob/f38e45016276218330480a40a5575b3d5523522e/ydb/library/yql/sql/v1/sql_translation.cpp#L4855) an AST of the query text were discarded previously. Early validation of the queries stored in views didn't work.